From 8b31f9c5f02e502f57a42610e230fdae43bfbf50 Mon Sep 17 00:00:00 2001
From: Kevin Thorne <kevint@aptitekk.com>
Date: Wed, 30 Jul 2014 19:43:47 -0600
Subject: [PATCH 085/101] Updated class path to use eclipse's default JDK

---
 TeamScreen/.classpath             | 44 +++++++++++++++----------------
 TeamScreen_AutoUpdater/.classpath | 16 +++++------
 2 files changed, 30 insertions(+), 30 deletions(-)

diff --git a/TeamScreen/.classpath b/TeamScreen/.classpath
index a8b87cd..983fb52 100644
--- a/TeamScreen/.classpath
+++ b/TeamScreen/.classpath
@@ -1,22 +1,22 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="lib"/>
-	<classpathentry kind="lib" path="lib/core.jar"/>
-	<classpathentry kind="lib" path="lib/java-remote-control-with-source.jar"/>
-	<classpathentry kind="lib" path="lib/jna-4.1.0.jar">
-		<attributes>
-			<attribute name="javadoc_location" value="http://twall.github.io/jna/4.1.0/"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="lib" path="lib/jna-platform-4.1.0.jar">
-		<attributes>
-			<attribute name="javadoc_location" value="http://twall.github.io/jna/4.1.0/"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="lib" path="lib/open-forum.jar"/>
-	<classpathentry kind="lib" path="lib/jintellitype-1.3.9.jar"/>
-	<classpathentry kind="lib" path="lib/commons-io-2.4.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/jre7"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="lib"/>
+	<classpathentry kind="lib" path="lib/core.jar"/>
+	<classpathentry kind="lib" path="lib/java-remote-control-with-source.jar"/>
+	<classpathentry kind="lib" path="lib/jna-4.1.0.jar">
+		<attributes>
+			<attribute name="javadoc_location" value="http://twall.github.io/jna/4.1.0/"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib/jna-platform-4.1.0.jar">
+		<attributes>
+			<attribute name="javadoc_location" value="http://twall.github.io/jna/4.1.0/"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="lib/open-forum.jar"/>
+	<classpathentry kind="lib" path="lib/jintellitype-1.3.9.jar"/>
+	<classpathentry kind="lib" path="lib/commons-io-2.4.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="output" path="bin"/>
+</classpath>
diff --git a/TeamScreen_AutoUpdater/.classpath b/TeamScreen_AutoUpdater/.classpath
index 9e641ba..f5837d1 100644
--- a/TeamScreen_AutoUpdater/.classpath
+++ b/TeamScreen_AutoUpdater/.classpath
@@ -1,8 +1,8 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/jre6"/>
-	<classpathentry kind="src" path="lib"/>
-	<classpathentry kind="lib" path="lib/zip4j_1.3.2.jar"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="src" path="lib"/>
+	<classpathentry kind="lib" path="lib/zip4j_1.3.2.jar"/>
+	<classpathentry kind="output" path="bin"/>
+</classpath>
-- 
2.39.5 (Apple Git-154)

